source/_integrations/broadlink.markdown

```diff
@@ -39,25 +39,28 @@ The entities are divided into three subdomains:
 
 ## Remote
 
-The `remote` entities allow you to control Broadlink universal remotes. You can learn IR codes, send the codes you have learned and send RF codes (use the base64 functionality). These entities are created automatically when you configure a device that has IR/RF capabilities.
+The `remote` entities are created automatically when you configure a universal remote. Use them to learn, store and send IR and RF commands.
 
-### Learning IR codes
+### Learning IR/RF codes
 
-Use the `remote.learn_command` service to learn IR codes.
+Use the `remote.learn_command` service to learn IR/RF codes.
 
 | Service data attribute | Optional | Description                          |
 | ---------------------- | -------- | ------------------------------------ |
 | `entity_id`            | no       | ID of the remote.                    |
 | `device`               | no       | Name of the device to be controlled. |
 | `command`              | no       | Names of the commands to be learned. |
+| `command_type`         | yes      | Type of the commands to be learned.  |
```

Comment:
Maybe we need to clarify the different types that can be used for this attribute?